src/Task/PhpCsFixerV2.php

```diff
@@ -91,6 +86,8 @@ public function run(ContextInterface $context)
             return $this->runOnAllFiles($context, $arguments);
         }
 
+        $arguments->add('--path-mode=intersection');
```

Comment:
Hi @keradus ,

It's not that I refuse to know it classes, it's about some configurations using the globally installed php-cs-fixer. For locally installed fixers it wouldn't be a problem. 
Instead of running the cli tool twice, it is also possible to make the path mode configurable again, but this time with a boolean that looks like `intersect_on_commit` with a default value of true. That would probably also fix this problem.